### ISSUE TITLE
Documentation is not up to date with v20.0.0 in README.md

### ISSUE THREAD

**Issue by jsyrjala** [Adyen/adyen-java-api-library#1026]
**Describe the bug**

README has this section https://github.com/Adyen/adyen-java-api-library#using-the-library
```
// Import the required classes
import com.adyen.Client;
import com.adyen.enums.Environment;
import com.adyen.service.Checkout;
import com.adyen.model.checkout.*;

// Setup Client and Service
Client client = new Client("Your X-API-KEY", Environment.TEST);
Checkout checkout = new Checkout(client);
```

however, class `com.adyen.service.Checkout` no longer exits in v20.0.0

**Comment by jillingk** [comment]
@jsyrjala,

Good catch! We switched the checkout class around but completely forgot to update this example. Thanks a lot for letting us know, it's much appreciated :)

Best, Jilling
Adyen

**Comment by jillingk** [comment]
Ditto for the webhookhandler, I'm gonna review the codesnippets to make sure they're still in line with our newest release.

**Comment by jsyrjala** [comment]
I guess also  `com.adyen.model.checkout.PaymentDonationRequestPaymentMethod` -> `com.adyen.model.checkout.CheckoutPaymentMethod` in https://github.com/Adyen/adyen-java-api-library#general-use-with-api-key
